client/my-sites/customer-home/stats-card/index.jsx

```diff
@@ -0,0 +1,144 @@
+	trafficData,
+	trafficStatsType,
+	trafficStatsQuery,
+} ) => {
+	const translate = useTranslate();
+
+	if ( hideStats ) {
+		return null;
+	}
+
+	return (
+		<Card className="stats-card">
+			{ siteId && (
+				<>
+					<QuerySiteStats
+						siteId={ siteId }
+						statType={ trafficStatsType }
+						query={ trafficStatsQuery }
+					/>
+					{ showInsights && (
+						<QuerySiteStats
+							siteId={ siteId }
+							statType={ insightsStatsType }
+							query={ insightsStatsQuery }
+						/>
+					) }
+				</>
+			) }
+			<CardHeading>{ translate( 'Stats at a glance' ) }</CardHeading>
+			<h6 className="stats-card__subheader">{ translate( 'Your site in the last week.' ) }</h6>
+			<div className="stats-card__data">
+				{ ! showInsights && (
+					<div className="stats-card__data-item">
+						<div className="stats-card__data-value">
+							{ trafficData?.views ? numberFormat( trafficData.views ) : '-' }
+						</div>
+						<div className="stats-card__data-label">{ translate( 'Views' ) }</div>
+					</div>
+				) }
+				{ ! showInsights && (
+					<div className="stats-card__data-item">
+						<div className="stats-card__data-value">
+							{ trafficData?.visitors ? numberFormat( trafficData.visitors ) : '-' }
+						</div>
+						<div className="stats-card__data-label">{ translate( 'Visitors' ) }</div>
+					</div>
+				) }
+				{ showInsights && (
+					<div className="stats-card__data-item">
+						<div className="stats-card__data-label">{ translate( 'Most popular day' ) }</div>
+						<div className="stats-card__data-value">{ insightsData?.day ?? '-' }</div>
+					</div>
+				) }
+				{ showInsights && (
+					<div className="stats-card__data-item">
+						<div className="stats-card__data-label">{ translate( 'Most popular hour' ) }</div>
+						<div className="stats-card__data-value">{ insightsData?.hour ?? '-' }</div>
+					</div>
+				) }
+			</div>
+			<a href={ `/stats/day/${ siteSlug }` }>{ translate( 'See all stats' ) }</a>
+		</Card>
+	);
+};
+
+const mapStateToProps = state => {
+	const siteId = getSelectedSiteId( state );
+	const siteSlug = getSelectedSiteSlug( state );
+	const isJetpack = isJetpackSite( state, siteId );
+	const isStatsModuleActive = isJetpackModuleActive( state, siteId, 'stats' );
+
+	const trafficStatsType = 'statsVisits';
+	const trafficStatsQuery = {
+		unit: 'week',
+		quantity: 1,
+		stat_fields: 'views,visitors',
+	};
+	const trafficStats = getSiteStatsNormalizedData(
+		state,
+		siteId,
+		trafficStatsType,
+		trafficStatsQuery
+	);
+	const trafficData = trafficStats && trafficStats.length ? trafficStats[ 0 ] : null;
+
+	const showInsights = trafficData && trafficData.views < 10;
+
+	const insightsStatsType = 'statsInsights';
+	const insightsStatsQuery = {};
+	const insightsData = getSiteStatsNormalizedData(
+		state,
+		siteId,
+		insightsStatsType,
+		insightsStatsQuery
+	);
+
+	const hideStats =
+		( isJetpack && ! isStatsModuleActive ) || ( showInsights && ! insightsData?.percent );
```

Comment:
Should we hide the card, when insightsData.percent is 0? This will commonly happen right after site launch

Reply:
That's the goal here. If there is no insights data or if insightsData.percent is 0 then `! insightsData?.percent` will be true which will effectively hide the card. I think it makes sense to hide it since there is no data we can display.